src/libs/actions/Policy.ts

```diff
@@ -277,6 +277,16 @@ function getPolicy(policyID: string | undefined): Policy | EmptyObject {
     return allPolicies[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`] ?? {};
 }
 
+/**
+ * Returns a primary policy for the user
+ */
+function getPrimaryPolicy(activePolicyID?: string): Policy | undefined {
+    const activeAdminWorkspaces = PolicyUtils.getActiveAdminWorkspaces(allPolicies);
+    const primaryPolicy: Policy | null | undefined = allPolicies?.[activePolicyID ?? ''];
```

Comment:
As we did not exclude personal policies while fetching primary policy, this resulted in #46705